.pre-commit-config.yaml

```
@@ -0,0 +1,92 @@
+repos:
+- repo: local
+  hooks:
+    - id: docformatter
+      name: Format docstrings with `docformatter`
+      language: system
+      types: [python]
+      require_serial: true
+      entry:  docformatter
+      args: [
+        --in-place, --wrap-summaries=88, --wrap-descriptions=87, --recursive
+      ]
+      files: ^(melloddy_predictor|tests|examples)/
+-   repo: local
```

There are too many spaces:
```suggestion
- repo: local
```